internal/sidekick/refresh.go

```diff
@@ -61,55 +61,13 @@ func loadDir(rootConfig *config.Config, output string) (*api.API, *config.Config
 	if config.General.SpecificationSource == "" {
 		return nil, nil, fmt.Errorf("must provide general.specification-source")
 	}
-	model, err := createModel(config)
+	model, err := parser.CreateModel(config)
 	if err != nil {
 		return nil, nil, err
 	}
 	return model, config, nil
 }
 
-func createModel(config *config.Config) (*api.API, error) {
-	var err error
-	var model *api.API
-	switch config.General.SpecificationFormat {
-	case "openapi":
-		model, err = parser.ParseOpenAPI(config.General.SpecificationSource, config.General.ServiceConfig, config.Source)
-	case "protobuf":
-		model, err = parser.ParseProtobuf(config.General.SpecificationSource, config.General.ServiceConfig, config.Source)
-	case "none":
-		return nil, nil
```

Comment:
I use this in storage control

https://github.com/googleapis/google-cloud-rust/blob/c329939e22fcaf342c3e58fc35cf5242feca0f2a/src/storage/src/control/generated/.sidekick.toml

Reply:
Argh, I searched for any usage but my grep-fu was not strong today. Fixed.